### PR TITLE
UI: work on CI run view, tweak c-bench landing page, misc

### PR DESCRIPTION
**Pull request by jgehrcke**
Quite a bit of UI work along ongoing initiatives/ideas. This patch focuses on the CI run view (showing details about the CI run, and also the results submitted as part of the run).

### CODE REVIEW

**Comment by jgehrcke** [comment]
:ship:  feedback appreciated!

File: conbench/templates/app.html [hunk 1]
```diff
@@ -91,21 +83,20 @@
                 <div class="modal-content">
                   <div class="modal-header">
                     <h4 class="modal-title">
-                      <i class="bi bi-trash3-fill"></i>
+                      <i class="bi bi-trash3-fill"></i> garbage or not? maybe it's art?
```
Comment (austin3dickey) [inline conbench/templates/app.html:86]: Haha, what does this mean?

image

File: conbench/templates/run.html [hunk 2]
```diff
@@ -1,18 +1,53 @@
 {% extends "app.html" %}
 {% import 'bootstrap/wtf.html' as wtf %}
 {% block app_content %}
-  <h4>CI run {{ run.id[:20] }}</h4>
+  <h1>CI run {{ run.id[:20] }}</h1>
+  <div class="mt-2">
+    Benchmarked code:
```
Comment (austin3dickey) [inline conbench/templates/run.html:6]: Nice!

File: conbench/templates/app.html (hunk 1 above)
Comment (jgehrcke) [reply conbench/templates/app.html:86]: :D In German we have the saying 

>  Ist das Kunst oder kann das Weg?

In English this is like "trash or treasure?" and here I took a freestyle approach. Let me know if that's too stupid / not good :).

Comment (austin3dickey) [reply conbench/templates/app.html:86]: No I like the whimsy. Software shouldn't take itself too seriously :)

**Pull request merged**